requirements.txt

```diff
@@ -32,4 +32,9 @@ jinja-partials==0.1.1
 hypothesis>=6.75.3
 tqdm==4.65.0
 pytest-xdist==3.3.1
-email-validator==2.1.0.post1
\ No newline at end of file
+email-validator==2.1.0.post1
```

Comment:
Hi @SabinaChita!

I am not sure why these are added, do you know? Did you add this on purpose?